pkg/reconciler/cmd/start.go

```diff
@@ -98,11 +102,6 @@ func Start(namespaced bool, managedNamespace string) {
 		logger.Fatalw("Unable to watch ConfigMaps", zap.Error(err))
 	}
 
-	// Watch Secrets with ResourceVersion changes, and enqueue owning InterStepBuffer key
-	if err := isbSvcController.Watch(&source.Kind{Type: &corev1.Secret{}}, &handler.EnqueueRequestForOwner{OwnerType: &dfv1.InterStepBufferService{}, IsController: true}, predicate.ResourceVersionChangedPredicate{}); err != nil {
```

Comment:
We don't watch the secrets any more, the down side is if somebody manually deleted the secrets, the controller will not immediately get notification for reconciliation, but every it will do it in re-sync every 20 mins. Considering manually touching the secret is a rare case, we can ignore it.